docs/getting_started.rst

```diff
@@ -137,9 +137,9 @@ The Python project-specific dependencies installed above will install ``nodeenv`
   npm install -g yarn
 
   # other required project dependencies
+  # omit the --force flag to speed up the process
   yarn install --force
 
-Note that the ``--force`` flag above can also be omitted to speed up the process.
```

Comment:
`--upgrade` flags above did not seem to add too much build time, but this one with `--force` seemed neverending, so I thought it better to add the note _before_ actually running the command.

Reply:
I would recommend just replacing the command with `yarn install` and skipping any mention of `--force`. In practice I never do that